js/main.js

```diff
@@ -108,7 +110,8 @@ async function main() {
         // watch for any selection changes - either grid or pile card display
         GACHA_VIEW_SETTING[0].addEventListener("change", updateGachaView);
         GACHA_VIEW_SETTING[1].addEventListener("change", updateGachaView);
-
+        // add or remove the shine animation on the cards to reduce the amount of animations there are
```

Comment:
(nit) it doesn't just control shine animation, so "add/ remove game animations" would be a more accurate comment. No comment at all would be OK too, the code is self-explanatory imo

Reply:
Sorry, the behaviour changed after our discussion today so I forgot to update this. I have removed it.